libraries/AP_Scripting/generator/src/main.c

```diff
@@ -673,7 +673,7 @@ void handle_operator(struct userdata *data) {
     error(ERROR_USERDATA, "Needed a symbol for the operator");
   }
 
-  enum operator_type operation;
+  enum operator_type operation = OP_ADD;
```

Comment:
Could this be done better with a NORETURN or whatever attribute on "error"?

Reply:
This change already went in apparently.